api_tests/src/community.spec.ts

```diff
@@ -380,16 +380,16 @@ test("User blocks instance, communities are hidden", async () => {
 test("Community follower count is federated", async () => {
   // Follow the beta community from alpha
   let community = await createCommunity(beta);
-  let community_id = community.community_view.community.actor_id;
-  let resolved = await resolveCommunity(alpha, community_id);
+  let communityActorId = community.community_view.community.actor_id;
```

Comment:
I believe standard rust casing for variables is snake_case.

Reply:
That's a typescript file, patrick!

Reply:
Doh!